dev/bitcoin/main.tf

```diff
@@ -39,6 +39,9 @@ resource "kubernetes_secret" "bitcoind_smoketest" {
     bitcoind_endpoint    = "bitcoind.${local.bitcoin_namespace}.svc.cluster.local"
     bitcoind_port        = 18443
     bitcoind_user        = "rpcuser"
+
+    bitcoind_onchain_rpcpassword = local.bitcoind_rpcpassword
```

Comment:
Should this be `local.bitcoind_onchain_rpcpassword` or something?

Reply:
reusing the same password